at_end2end_test/test/notify_test.dart

```diff
@@ -108,42 +105,29 @@ void main() {
         // Setting the AtClientManager instance to current atsign
         await AtClientManager.getInstance().setCurrentAtSign(
             currentAtSign, namespace, TestUtils.getPreference(currentAtSign));
-
-        var epochMillsNow = DateTime.now().millisecondsSinceEpoch;
         var notificationResult = await AtClientManager.getInstance()
             .notificationService
             .notify(input);
         expect(notificationResult.notificationStatusEnum,
             NotificationStatusEnum.delivered);
 
-        // Setting the AtClientManager instance to sharedWith atsign
-        await AtClientManager.getInstance().setCurrentAtSign(sharedWithAtSign,
-            namespace, TestUtils.getPreference(sharedWithAtSign));
-        // Getting the notification via the monitor command
-        var streamController = StreamController<AtNotification>();
-        var isNotificationReceived = false;
-        var monitorForNotification = MonitorForNotification(
-            notificationResult.notificationID,
-            sharedWithAtSign,
-            epochMillsNow,
-            streamController);
-        await monitorForNotification.init();
-        streamController.stream.listen((atNotification) async {
-          isNotificationReceived = true;
-          var response =
-              await NotificationResponseTransformer().transform(Tuple()
-                ..one = atNotification
-                ..two = (NotificationConfig()..shouldDecrypt = true));
-          expect(response.key, expectedOutput);
-        });
-        while (!isNotificationReceived) {
-          await Future.delayed(Duration(milliseconds: 100));
-        }
-      });
+            await AtClientManager.getInstance().setCurrentAtSign(
+                sharedWithAtSign,
+                namespace, TestUtils.getPreference(sharedWithAtSign));
+            var atNotification = await AtClientManager
+                .getInstance()
+                .notificationService
+                .fetch(notificationResult.notificationID);
+            atNotification.isEncrypted = input.atKey.metadata!.isEncrypted;
+            await NotificationResponseTransformer().transform(Tuple()
+              ..one = atNotification
+              ..two = (NotificationConfig()
+                ..shouldDecrypt = input.atKey.metadata!.isEncrypted!));
+            expect(atNotification.id, notificationResult.notificationID);
+            expect(atNotification.key, '$whomToNotify:$notifyText');
```

Comment:
We already know the expected output, please change to `expect(atNotification.key, expectedOutput);`

Reply:
Updated with the expectedOutput